README.md

```diff
@@ -166,6 +169,10 @@ Simply use this parameter to set it. The host record will be created in Conjur.
 Raw (unencoded) Conjur token. This is usually only useful for testing.
 Must be `Sensitive` if supported.
 
+##### `version`
+Conjur API version. Defaults to 4.
+Set to 5 if you're using the open source Conjur edition.
```

Comment:
I think you misunderstood the implication here ;)
It doesn't say 'only'. But if you're new with Conjur and you have open source you might not know it's 5; especially since it's 0.1.0.

Reply:
It just says `version = 5 if opensource?`.

Reply:
(Though if you misunderstood it, others might too. Can you think of better wording?)

Reply:
I think we need an example of using both in the README. it's confusing now. Can you add that Rafal?

Reply:
Sure. Maybe modify all examples to opensource and add a separate section for Enterprise?